packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -526,6 +526,9 @@ class Overlay extends StatefulWidget {
 class OverlayState extends State<Overlay> with TickerProviderStateMixin {
   final List<OverlayEntry> _entries = <OverlayEntry>[];
 
+  /// all OverlayEntry
```

Comment:
This doc does not meet our style guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc